docs/manual/output.md

```diff
@@ -99,6 +119,32 @@ $ mv output.{hdf5,lh5}
 
 :::
 
+### Reshaping output tables
+
+For the LH5 output and _Germanium_ or _Scintillator_ output we implemented a
+"reshaping" of the output tables. This groups together rows in the same output
+table that have the same simulated Geant4 evtid and also with times with the
+user defined time window (more later). In this way the rows of the output table
+represent physical interactions in each output table.
```

Comment:
It would be useful here to show the table (or just a column) repr before and after the reshaping. like:
```
[1,
 2,
 3,
 ...,
]
```
and then
```
[
  [1, 2, ...],
  [4]
]
```
or whatever looks better. you can just put the awkward `.show()` printout